src/Controls/tests/DeviceTests/Elements/FlyoutPage/FlyoutPageTests.iOS.cs

```diff
@@ -66,33 +60,31 @@ public async Task FlyoutPageTakesIntoAccountSafeAreaByDefault()
 		[InlineData(true)]
 		public async Task DetailsViewPopOverLayoutIsCorrectForIdiom(bool isRtl)
 		{
-			if (isRtl && System.OperatingSystem.IsIOSVersionAtLeast(17))
-			{
-				//skip till we figure the 1 pixel issue 
```

Comment:
So this comment maybe should be kept for iOS17? the current runs aren't running iOS17

Reply:
No, we should just turn on iOS 17 and fix the issue.

Reply:
When I ran this locally against iOS17 it passed after the fixes I made to the issue down below